relay-server/src/services/store.rs

```diff
@@ -299,7 +299,13 @@ impl StoreService {
                     self.produce_span(scoping, start_time, event_id, retention, item)?
                 }
                 other => {
-                    relay_log::error!("StoreService received unexpected item type: {other}");
+                    let event_type = event_item.as_ref().map(|item| item.ty());
+                    relay_log::error!(
+                        tags.project_key = %scoping.project_key,
+                        tags.event_type = ?event_type,
+                        envelope_items = envelope.items().len(),
```

Comment:
Since `envelope_items` is in the context, it's cheap to add the types of the items in the envelope, in case that's useful.

Reply:
Not sure what you mean, add the envelop items to `extra`?

Reply:
Adding a list of the types of the envelope's items to the event's extra/context.